accounts/services/distributed/__init__.py

```diff
@@ -116,6 +149,17 @@ def create_session(user_data: UserData) -> SessionData:
     return current_session().create_session(user_data)
 
 
+@wraps(RedisSession.get_session)
+def get_session(session_id: str) -> Optional[Any]:
+    """
+    Invalidates a session in the key-value store.
+
+    Parameters
+    ----------
+    session_id : str
+    """
+    return current_session().get_session(session_id)    
+
 @wraps(RedisSession.delete_session)
```

Comment:
Style thing: be sure to have two blank lines before a top-level class or function (including decorators)